c_src/CMakeLists.txt

```diff
@@ -12,6 +12,15 @@ include(ASan)
 IF (APPLE)
   set(CMAKE_MODULE_LINKER_FLAGS "-flat_namespace -undefined suppress")
   set(CMAKE_MACOSX_RPATH 1)
+  if (NOT DEFINED ENV{SDKROOT})
+    execute_process(COMMAND
+      xcrun --show-sdk-path
+      OUTPUT_VARIABLE APPLE_SDK_ROOT
+      )
```

Comment:
Argh. Hanging indent here. Not to my most excellent standards! Still compiles.

Reply:
I often wish there was a `cmakefmt` command with zero config options

Reply:
> Not to my most excellent standards
because it's a copy of the way I indent cmake for no other reason than that's how emacs indents it!